glide.yaml

```diff
@@ -1,29 +1,29 @@
+# Update deps with `glide update --strip-vendor`
+
 package: github.com/openshift/cluster-authentication-operator
 import:
 - package: k8s.io/apimachinery
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/api
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/apiserver
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/client-go
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
+- package: k8s.io/code-generator
+  version: kubernetes-1.16.0
 - package: k8s.io/apiextensions-apiserver
-  repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
```

Comment:
Can we safely turn this into the regular apiextensions-apiserver now?

Reply:
My assumption was that the upstream carries for 1.14 are included in 1.16, and the 1.16 branch of `https://github.com/openshift/kubernetes-apiextensions-apiserver` appeared to  have no carries.

Reply:
sounds good, then